test/bdd/webhook/main.go

```diff
@@ -64,27 +52,14 @@ func checkTopics(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func messages(w http.ResponseWriter, r *http.Request) {
-	msg, err := ioutil.ReadAll(r.Body)
-	if err != nil {
-		w.WriteHeader(http.StatusBadRequest)
-	}
-
-	logger.Infof("received generic msg event")
-
-	topics <- msg
-}
-
 func main() {
 	port := os.Getenv("WEBHOOK_PORT")
 	if port == "" {
 		panic("port to be passed as ENV variable")
 	}
 
 	router := mux.NewRouter().StrictSlash(true)
-	router.HandleFunc(connectionsPath, connections).Methods(http.MethodPost)
+	router.HandleFunc("/", connections).Methods(http.MethodPost)
```

Comment:
May be we can change the name of handler from `connections` to `handleNotification` or something like that. With this refactor, the api will handle generic notification and not just related to connections/didexchange functionality.